node-data/src/ledger.rs

```diff
@@ -94,6 +94,7 @@ pub struct SpentTransaction {
     pub inner: Transaction,
     pub block_height: u64,
     pub gas_spent: u64,
+    pub economic_gas_spent: u64,
```

Comment:
Ditto.
We should agree on what this field is, and furthermore we should be able to derive back the whole economic info.

So I suggest to include the whole economic mode (to know if we are talking about allowance, charge, or whatever) and to be sure we're just using this field as Gas Points and not mixing it with the charged Fee (gas*price)

Reply:
let's consider it, on the other hand it is kind of obvious, if `economic_gas_spent` is zero, it is an allowance, if it is greater than `gas_spent`, it is a charge

Reply:
I'm a big fan of all information being readily available, and not just "derivable" downstream. I agree with Seppia here.

If you think including the whole of the `EconomicMode` somehow too "heavy handed" , then the other option is a function computing that info.